src/main/features/claim/form/models/claimAmountRow.ts

```diff
@@ -38,13 +38,27 @@ export class ClaimAmountRow {
     return new ClaimAmountRow(undefined, undefined)
   }
 
+  static translateToNumber (input?: any): any {
```

Comment:
There is already `toNumberOrUndefined` function which looks like better place to put that change if I understand this correctly that it is about handling comma thousand separators in UK localised systems.

Fixing problem in just a single place is very short sighted solution.